babel/dates.py

```diff
@@ -707,6 +754,9 @@ def format_skeleton(skeleton, datetime=None, tzinfo=None, locale=LC_TIME):
     u'dim. 1 avr.'
     >>> format_skeleton('MMMEd', t, locale='en')
     u'Sun, Apr 1'
+    >>> format_skeleton('yMMd', t, locale='fi')  # yMMd is not in the Finnish locale; yMd gets used
```

Comment:
For demonstration purposes, I think it would be good to also have this call again, except with fuzzy=False